app/helpers/groups_helper.rb

```diff
@@ -20,6 +20,14 @@ def new_event_button
     end
   end
 
+  def export_events_ical_button
```

Comment:
Currently, there is no spec-file for the `GroupsHelper`, I would create one in `spec/helpers/groups_helper_spec.rb` and take inspiration from https://github.com/hitobito/hitobito/blob/master/spec/helpers/filter_navigation/people_spec.rb, which also handles the cancan-mocking. Does this help?